src/components/Portal/Portal.tsx

```diff
@@ -67,8 +68,14 @@ export class Portal extends React.PureComponent<PortalProps, State> {
   }
 
   render() {
+    const childrenMarkup = (
+      <ButtonGroupContext.Provider value={undefined}>
```

Comment:
@BPScott I was thinking this could be a valid option. I debated adding this to popover because additional buttons would always be inside a popover, however, new overlays could come at play one day and this will guard against these as well.

Reply:
Nice trick! I'll forget all about it in a few weeks so it'd be worth adding a comment about why this is here